lib/googleauth/compute_engine.rb

```diff
@@ -68,21 +67,29 @@ def compute_id_token_uri
 
         # Detect if this appear to be a GCE instance, by checking if metadata
         # is available.
-        def on_gce? options = {}
-          # TODO: This should use google-cloud-env instead.
-          c = options[:connection] || Faraday.default_connection
-          headers = { "Metadata-Flavor" => "Google" }
-          resp = c.get compute_check_uri, nil, headers do |req|
-            req.options.timeout = 1.0
-            req.options.open_timeout = 0.1
+        def on_gce? options = {}, reload = false # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
That argument is only there because memoist would add it automatically.

If it's made into a keyword argument, then backward compatibility is broken, which I don't mind, but then I'd rather just not add it at all.